k8sutils/redis-cluster.go

```diff
@@ -184,3 +185,21 @@ func getRedisLabels(name, setupType, role string) map[string]string {
 		"role":             role,
 	}
 }
+
+// generateSidecars generates Redis sidecar informations
+func generateClusterSidecars(cr *redisv1beta1.RedisCluster /*cr *redisv1beta1.Redis*/) []sidecarParameters {
+	sidecars := []sidecarParameters{}
+
+	if cr.Spec.Sidecars != nil {
+		for i := 0; i < len(*cr.Spec.Sidecars); i++ {
+			sidecars = append(sidecars, sidecarParameters{
+				Name:            (*cr.Spec.Sidecars)[i].Name,
+				Image:           (*cr.Spec.Sidecars)[i].Image,
+				ImagePullPolicy: (*cr.Spec.Sidecars)[i].ImagePullPolicy,
+				Resources:       (*cr.Spec.Sidecars)[i].Resources,
+				EnvVars:         (*cr.Spec.Sidecars)[i].EnvVars,
+			})
+		}
+	}
```

Comment:
```suggestion
		for _, sidecar := range *cr.Spec.Sidecars {
			sidecars = append(sidecars, sidecarParameters{
				Name:            sidecar.Name,
				Image:           sidecar.Image,
				ImagePullPolicy: sidecar.ImagePullPolicy,
				Resources:       sidecar.Resources,
				EnvVars:         sidecar.EnvVars,
			})
		}
	}
```

A bit cleaner that way and prevent having to resolve slice values for each attribute.